station/waterfall.py

```diff
@@ -1,23 +1,25 @@
 # This file is heavily based on waterfall.py from satnogs-client.
 
+import matplotlib.pyplot as plt
 import matplotlib
 import numpy as np
 import sys
```

Comment:
Does the import order check by the linter? PEP8 recommends sorting the imports alphabetically and split them into three groups: stdlib imports, external imports, and internal imports. [Source](https://peps.python.org/pep-0008/#imports)

> Imports should be grouped in the following order:
> 
> Standard library imports.
> Related third party imports.
> Local application/library specific imports.
>
>You should put a blank line between each group of imports.

Reply:
Fixed this specific item. But there are lots of others that `pylint` still complains about. That's why we have 7.0 threshold, not 10.0. We'll get there one day...